styles/file-drop.css

```diff
@@ -16,3 +16,22 @@
     background-color: rgba(0,155,0,0.5);
   }
 }
+
+.primary-track-table-drag-over {
```

Comment:
@ahdinosaur starting to need some styling. we should setup that infrastructure next time around